orion/pkg/tests_integration/user_integration_test.go

```diff
@@ -0,0 +1,180 @@
+package integration_tests
+
+import (
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/domains"
+	"github.com/stretchr/testify/assert"
+)
+
+// Test: Create 3 Users and GetAll()
+func Test_CreateUsers(t *testing.T) {
+	resetTable(t, domains.TABLE_USERS)
+
+	user1 := createUser(
+		"John",
+		"Smith",
+		"Middle1",
+		"john_smith@example.com",
+		"555-555-0100",
+		false,
+		2,
+	)
+	user2 := createUser(
+		"Bob",
+		"Joe",
+		"Middle2",
+		"bob_joe@example.com",
+		"555-555-0101",
+		false,
+		1,
+	)
+	user3 := createUser(
+		"Foo",
+		"Bar",
+		"Middle3",
+		"foobar@example.com",
+		"555-555-0102",
+		false,
+		1,
+	)
+	body1 := createJsonBody(&user1)
```

Comment:
Again note how we pass the pointers of these structs and not the structs themselves when Json Marshaling